docs: add the manager's RPC client certificate entry to the guide (#2052)
lablup/backend.ai: docs/install/install-from-package/install-manager.rst

   $ pip install -U backend.ai-manager==${BACKEND_PKG_VERSION}

> For the first time, you need to generate rpc keypair. This keypair is used for authentication and encryption to our manager-to-agent RPC connections.

Reviewer: My suggestion:
> You may need to generate an RPC keypair for the first setup. This key pair is used for authentication and encryption in our manager-to-agent RPC connections. Even if you don't intend to encrypt the RPC channel, the key pair should still be present.

.. code-block:: console

   $ cd "${HOME}/manager"
   $ mkdir -p fixtures/manager

>    $ backend.ai mgr generate-rpc-keypair fixtures/manager 'manager'

Reviewer: This is a minor one, but I prefer to generate the key pair under `fixtures/` not `fixtures/manager/`. It already is evident the `fixtures` will contain manager-only configurations here.

Author: I'll change `fixtures/manager` to `fixtures/`

   $ pip install -U backend.ai-manager==${BACKEND_PKG_VERSION}

> You may need to generate an RPC keypair for the first setup. This key pair is used for authentication and encryption in our manager-to-agent RPC connections.
> Even if you don't intend to encrypt the RPC channel, the key pair should still be present.

Reviewer 2: ```suggestion
You may need to generate an RPC keypair for the first setup. This keypair is used for authentication and encryption in our manager-to-agent RPC connections.
Even if you don't want to encrypt the RPC channel, you still need to have a keypair.
```

> You may need to generate an RPC keypair for the first setup. This key pair is used for authentication and encryption in our manager-to-agent RPC connections.
> Even if you don't intend to encrypt the RPC channel, the key pair should still be present.

Reviewer 2: ```suggestion
You may need to generate an RPC keypair for the first setup. This keypair is used for authentication and encryption in our manager-to-agent RPC connections.
Even if you don't want to encrypt the RPC channel, you still need to have a keypair.
```

Author: modified it.

.. code-block:: console

   $ cd "${HOME}/manager"
   $ mkdir fixtures
   $ backend.ai mgr generate-rpc-keypair fixtures 'manager'

   2024-04-23 12:06:31.913 INFO ai.backend.manager.cli [21024] Generating a RPC keypair...
    Public Key: >B-mF}N{WygT92d&=Kceix$7cWzg!dT])rIc39=S (stored at fixtures/manager.key)
    Secret Key: g.4&?*b&0#oRRC9?DMO[SUXikjKZ7nYj!bzFJN92 (stored at fixtures/manager.key_secret)

> You generated an RPC keypair for the manager. The public key is stored in ``fixtures/manager.key`` and the secret key is stored in ``fixtures/manager.key_secret``.

Reviewer 2: ```suggestion
You have generated an RPC keypair for the manager. The public key is stored in ``fixtures/manager.key`` and the secret key is stored in ``fixtures/manager.key_secret``.
```

Author: Also, I modified the description.

   ssl-enabled = false

>    heartbeat-timeout = 30.0

Reviewer: Let's update `heartbeat-timeout` to `40.0` since the heart beat period from agent is increased in 23.09.

Author: fixed it.

>    rpc-auth-manager-keypair = "/home/bai/manager/fixtures/manager/manager.key_secret"

Reviewer: No need to include `/home/bai/manager`. It would be enough to just write `rpc-auth-manager-keypair = fixtures/manager.key_secret`.

Author: [The document](https://docs.backend.ai/en/latest/install/install-from-package/install-manager.html#local-configuration) specifies the location of `manager.toml` as `${HOME}/.config/backend.ai/manager.toml`

So, I modify as follows according to the above modifications. 
rpc-auth-manager-keypair =  "/home/bai/manager/fixtures/manager.key_secret"

Reviewer: Sometimes, we cannot install the manager under the `bai` account. Depending on the situation, a different OS account may be used. In such cases, it's better to remove hardcoded references if possible. By specifying `fixtures/manager.key_secret`, the `fixtures` directory will be recognized as a relative path under the directory where the manager is installed. This allows for a more generalized configuration.

.. code-block:: console

   $ vfolder_host_val='{"bai-m1:local": ["create-vfolder", "modify-vfolder", "delete-vfolder", "mount-in-session", "upload-file", "download-file", "invite-others", "set-user-specific-permission"]}'

>    $ docker exec -it $(docker ps | grep backendai-pg | awk '{print $1}') psql -U postgres -d backend \

Reviewer 3: Whenever possible, we should prefer the exact reference mechanism.
Here, we could use `docker compose exec -it backendai-pg psql -U postgres -d backend` to exactly refer to the target postgres container of the current compose stack, as long as the compose stack is initialized via the cwd's `docker-compose.yml`. Otherwise, we should specify `-p` and/or `-f` options explicitly to the `docker compose` command.

Reviewer 3: There is no need to take care of the "not beautiful" container naming scheme of Docker Compose -- it's just an auto-generated one. We should use the identity of compose stack and the service name to refer the actual container, instead of their auto-generated names.
